test/spec/bidmanager_spec.js

```diff
@@ -264,7 +264,7 @@ var bidmanager = require('../../src/bidmanager');
                         }
                 };
 
-                var expected = {"hb_bidder":  bidderCode, "hb_adid": adId,"hb_pb": 15.0 };
```

Comment:
Need to change the (test description](https://github.com/prebid/Prebid.js/pull/295/files#diff-f9d389eae2d418abaaaabd456eee8a9cR213) since now it _should_ always inherit standard.